nginx_ingress_controller/datadog_checks/nginx_ingress_controller/nginx_ingress_controller.py

```diff
@@ -20,28 +55,7 @@ def __init__(self, name, init_config, instances):
                 "nginx_ingress": {
                     'prometheus_url': 'http://localhost:10254/metrics',
                     'namespace': 'nginx_ingress',
-                    'metrics': [
-                        # nginx metrics
-                        {'nginx_ingress_controller_nginx_process_connections': 'nginx.connections.current'},
-                        {'nginx_ingress_controller_nginx_process_connections_total': 'nginx.connections.total'},
-                        {'nginx_ingress_controller_nginx_process_requests_total': 'nginx.requests.total'},
-                        # nginx process metrics
-                        {'nginx_ingress_controller_nginx_process_num_procs': 'nginx.process.count'},
-                        {'nginx_ingress_controller_nginx_process_read_bytes_total': 'nginx.bytes.read'},
-                        {'nginx_ingress_controller_nginx_process_write_bytes_total': 'nginx.bytes.write'},
-                        {'nginx_ingress_controller_nginx_process_cpu_seconds_total': 'nginx.cpu.time'},
-                        {'nginx_ingress_controller_nginx_process_resident_memory_bytes': 'nginx.mem.resident'},
-                        {'nginx_ingress_controller_nginx_process_virtual_memory_bytes': 'nginx.mem.virtual'},
-                        # controller metrics
-                        {'nginx_ingress_controller_success': 'controller.reload.success'},
-                        {'nginx_ingress_controller_ingress_upstream_latency_seconds': 'controller.upstream.latency'},
-                        {'nginx_ingress_controller_response_duration_seconds': 'controller.response.duration'},
-                        {'nginx_ingress_controller_requests': 'controller.requests'},
-                        {'process_cpu_seconds_total': 'controller.cpu.time'},
-                        {'process_resident_memory_bytes': 'controller.mem.resident'},
-                        {'process_virtual_memory_bytes': 'controller.mem.virtual'},
-                    ],
-                    'send_histograms_buckets': True,
```

Comment:
Will removing this default modify behavior for existing users?

Reply:
This option is default true, this was just added in a PR that wasn't released yet.

Reply:
Ok, nice!